tests/unit/components/choropleth.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/vue";
+import { describe, expect, test } from "vitest";
+import { mockPinia } from "../mocks/mockPinia";
+import Choropleth from "../../../src/components/Choropleth.vue";
+
+const store = mockPinia();
+const renderComponent = async () => {
+    return render(Choropleth, {
+        global: {
+            plugins: [store],
+            stubs: {
+                LMap: true,
+                LTileLayer: true,
+                LGeoJson: true
+            }
+        }
+    });
+};
+
+describe("Choropleth", () => {
+    test("renders as expected", async () => {
+        await renderComponent();
+        const tileLayer = await screen.findByTestId("tile-layer");
```

Comment:
no need to change but should be able to do getBy and not await...

Reply:
Yeah, because in this case I don't actually need to wait for the data to be available (for loading to be false..)